client/signup/config/flows.js

```diff
@@ -130,7 +130,9 @@ const Flows = {
 	filterFlowName,
 	filterDestination,
 
-	defaultFlowName: abtest( 'improvedOnboarding' ) || 'main',
+	defaultFlowName: config.isEnabled( 'signup/atomic-store-flow' )
```

Comment:
Shouldn't this be `'signup/onboarding-flow'`? :D

Reply:
> Shouldn't this be 'signup/onboarding-flow'?

Ah, yeah, I was... ah, just testing your eyesight :trollface: 

(Thanks btw)